app/components/ui/tale-tab-metadata/template.hbs

```diff
@@ -0,0 +1,91 @@
+<div class="ui form">
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Title</label>
+    <div class="field thirteen wide column">
+        {{input value=model.tale.title placeholder="Enter a title..." readonly=cannotEditTale}}
+    </div>
+  </div>
+  
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Authors</label>
+    <div class="field thirteen wide column">
+        {{input value=model.tale.authors placeholder="Enter the author's name..." readonly="true"}}
+    </div>
+  </div>
+  
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Category</label>
+    <div class="field thirteen wide column">
+        {{input value=model.tale.category placeholder="Choose a category..."  readonly=cannotEditTale}}
+    </div>
+  </div>
+  
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Environment</label>
+    <div class="field thirteen wide column">
+      <div class='ui icon selection dropdown {{if (eq environments.length 0) "loading"}}' style='width:100%'>
+        <input name='TaleImageId' type='hidden' onchange={{action "setTaleEnvironment" value="target.value"}}>
+        <i class='dropdown icon'></i>
+        <div class='default text'> Choose an environment...</div>
+        <div class='menu'>
+          {{#each environments as |env|}}
+            <div class='item' data-value={{env._id}}>
+              <img class='ui image' src={{env.icon}} style='height:28px;width:28px;'> {{env.name}}
+            </div>
+          {{/each}}
+        </div>
+      </div>
+    </div>
+  </div>
+  
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Date Created</label>
+    <div class="field thirteen wide column">
+        {{input value=model.tale.created readonly="true"}}
```

Comment:
@bodom0015 according to the mockups, editable fields should be `input` and `select` elements, but readonly ones should just be labels, not to confuse the user as to suggest that they can be later be enabled for edition under a condition that might change.
<img width="2550" alt="screen shot 2018-12-06 at 10 16 26 am" src="https://user-images.githubusercontent.com/20361247/49593148-df0a0880-f940-11e8-947c-21c8a69aa165.png">

Reply:
The current mockups, including the screenshot you posted here, do not include **any** information about which fields are editable and which are not. Unless you are seeing a different picture than I am?

Reply:
Changed readonly inputs to `<span/>`